Docs/source/Software/Tasks.rst

```diff
@@ -0,0 +1,41 @@
+********************************
+Real Time Operating System Tasks
+********************************
+
+Fault State Task: Manthan Upadhyaya
+===================================
+
+The Fault State Task is called when a fault condition is set off in the BPS. 
+These fault conditions can be found here(place link to fault conditions).
+
+Process:
+    1) All other tasks are not allowed to run unless they have are utilized sempahore. The other tasks will run until they post that semaphore and then they are prevented from running again. This occurs internally through the RTOS.
+
+    2) The contactor is turned off.
+    
+    3) All the fans are set to maximum speed.
+    
+    4) The proper LED's are turned on and off. When the fault LED is turned on, the strobelight turns on.
+    
+    5) The fault condition is logged into the EEPROM.
+    
+    6) A message is sent along the CAN to any other system that might be operating still.
```

Comment:
The BPS Display board is the system that we expect to receive the CAN message. This will alert the driver that the BPS has faulted